### PR TITLE
Add a releasing doc.

### PR DESCRIPTION
**Pull request by dlorenc**
Ref #130

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Steps to Release Minikube
+
+## Create a Release Notes PR
+
+Assemble all the meaninful changes since the last release into the CHANGELOG.md file.
```
Comment (luxas) [inline RELEASING.md]: typo: meaningful

File: RELEASING.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+Run this command:
+```shell
+make integration
+```
+Investigate and fix any failures.
+
+## Tag the Release
+
+Run a command like this to tag it locally: ```git tag -a v0.2.0 -m "0.2.0 Release"```.
```
Comment (luxas) [inline RELEASING.md]: Only two simple ` works

**Comment by luxas** [comment]
nits only, LGTM

**Comment by luxas** [comment]
notify when you've updated the pr

**Comment by dlorenc** [comment]
Just pushed, you can see it here: https://github.com/dlorenc/minikube/blob/release/RELEASING.md

**Comment by luxas** [comment]
LGTM

**Pull request merged**